android/app/src/main/java/co/ello/ElloApp/MainActivity.java

```diff
@@ -391,7 +402,35 @@ public void onActionClicked(Snackbar snackbar) {
         registerReceiver(pushReceivedReceiver, new IntentFilter(ElloPreferences.PUSH_RECEIVED));
     }
 
+    private void setupRatePrompt(){
+        AppRate.with(this)
+            .setInstallDays(7)
+            .setLaunchTimes(3)
+            .setRemindInterval(7)
+            .setShowLaterButton(true)
+            .setDebug(true)
```

Comment:
Could we control it with an environment variable? Not sure if it's likely to ever be needed in the future or if it's a one-and-done sort of thing.